.github/CONTRIBUTING.md

```diff
@@ -82,11 +86,12 @@ git push origin main
 ```
 
 ##### Step 2.2: Create a feature branch
+
 - Create an issue on [github](https://github.com/open-mmlab/mmocr)
 
 - Create a feature branch
--
-  ```bash
+
+- ```bash
```

Comment:
In CONTRIBUTING.md, among all the bash blocks, only this one has a `-` ahead. Considering removing it?